spec/std/yaml/yaml_pull_parser_spec.cr

```diff
@@ -95,5 +106,11 @@ module YAML
         end
       end
     end
+
+    assert_raw %(hello)
+    assert_raw %("hello"), %(hello)
+    assert_raw %(["hello"])
+    assert_raw %(["hello","world"])
+    assert_raw %({"hello":"world"})
```

Comment:
It's not about parsing, it's that the correct string is serialized, since you do manual string building?